packages/origin-backend-app/src/origin-app.module.ts

```diff
@@ -79,10 +82,13 @@ export class OriginAppModule {
                 IssuerModule,
                 OrganizationModule,
                 UserModule,
-                CqrsModule
+                CqrsModule,
+                RegistrationModule
```

Comment:
@kosecki123 @JosephBagaric 
This is crazy, `RegistrationModule` is part of `IRECOrganizationModule`, but if I remove this line the whole thing breaks:
```
Nest can't resolve dependencies of the CertificateApprovedHandler (MailService, UserService, ?, QueryBus, ModuleRef). Please make sure that the argument RegistrationService at index [2] is available in the OriginAppModule context.
@energyweb/origin-backend-app: Potential solutions:
@energyweb/origin-backend-app: - If RegistrationService is a provider, is it part of the current OriginAppModule?
@energyweb/origin-backend-app: - If RegistrationService is exported from a separate @Module, is that module imported within OriginAppModule?
```

Reply:
Works as expected: "The module encapsulates providers by default. This means that it's impossible to inject providers that are neither directly part of the current module nor exported from the imported modules. Thus, you may consider the exported providers from a module as the module's public interface, or API"

Reply:
In this example, `IRECOrganizationModule` is probably overhead because it just import a single module.